packages/plugins/trpc/tests/projects/t3-trpc-v10/package.json

```diff
@@ -20,10 +20,14 @@
     "@trpc/next": "^10.43.6",
     "@trpc/react-query": "^10.43.6",
     "@trpc/server": "^10.43.6",
+    "@zenstackhq/language": "file:../../../../../../.build/zenstackhq-language-2.4.1.tgz",
```

Comment:
Please exclude this file from the PR. I know it's awkward that whenever the test run this file is updated ... but it should be excluded.